recipes/bwise/build.sh

```diff
@@ -1,7 +1,29 @@
 #!/bin/bash
 
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+
 cd src;
-make;
+g++ -o n50.o -c N50.cpp -Wall  -Ofast -std=c++11  -flto -pipe -funit-at-a-time  -Wfatal-errors -fopenmp
+g++ -o n50 n50.o -flto -fopenmp
+g++ -o sequencesToNumbers.o -c sequencesToNumbers.cpp -Wall  -Ofast -std=c++11  -flto -pipe -funit-at-a-time  -Wfatal-errors -fopenmp
+g++ -o sequencesToNumbers sequencesToNumbers.o -flto -fopenmp
+g++ -o numbersToSequences.o -c numbersToSequences.cpp -Wall  -Ofast -std=c++11  -flto -pipe -funit-at-a-time  -Wfatal-errors -fopenmp
+g++ -o numbersToSequences numbersToSequences.o -flto -fopenmp
+g++ -o numbersFilter.o -c numbersFilter.cpp -Wall  -Ofast -std=c++11  -flto -pipe -funit-at-a-time  -Wfatal-errors -fopenmp
+g++ -o numbersFilter numbersFilter.o -flto -fopenmp
+g++ -o simulator.o -c simulator.cpp -Wall  -Ofast -std=c++11  -flto -pipe -funit-at-a-time  -Wfatal-errors -fopenmp
+g++ -o simulator simulator.o -flto -fopenmp
+g++ -o path_counter.o -c path_counter.cpp -Wall  -Ofast -std=c++11  -flto -pipe -funit-at-a-time  -Wfatal-errors -fopenmp
+g++ -o path_counter path_counter.o -flto -fopenmp
+g++ -o maximal_sr.o -c Maximal_SR.cpp -Wall  -Ofast -std=c++11  -flto -pipe -funit-at-a-time  -Wfatal-errors -fopenmp
+g++ -o maximal_sr maximal_sr.o -flto -fopenmp
+g++ -o crush_bulle.o -c crush_bulle.cpp -Wall  -Ofast -std=c++11  -flto -pipe -funit-at-a-time  -Wfatal-errors -fopenmp
+g++ -o crush_bulle crush_bulle.o -flto -fopenmp
+g++ -o path_to_kmer.o -c path_to_kmer.cpp -Wall  -Ofast -std=c++11  -flto -pipe -funit-at-a-time  -Wfatal-errors -fopenmp
+g++ -o path_to_kmer path_to_kmer.o -flto -fopenmp
```

Comment:
Why are you side-stepping the Makefile? I'm not sure about this. Since you are upstream as well IIRC, maybe the Makefile should be fixed with a new version?

Anyhow, your zlib problem arises from not telling the linker about the library. You want something like this:

```
g++ -o path_to_kmer path_to_kmer.o -flto -fopenmp "-L${PREFIX}/lib"  -lzlib
```